KeeTrayTOTP/FormAbout.cs

```diff
@@ -51,16 +51,8 @@ internal string AssemblyTitle
         {
             get
             {
-                object[] attributes = Assembly.GetExecutingAssembly().GetCustomAttributes(typeof(AssemblyTitleAttribute), false);
-                if (attributes.Length > 0)
-                {
-                    var titleAttribute = (AssemblyTitleAttribute)attributes[0];
-                    if (titleAttribute.Title != string.Empty)
-                    {
-                        return titleAttribute.Title;
-                    }
-                }
-                return System.IO.Path.GetFileNameWithoutExtension(Assembly.GetExecutingAssembly().CodeBase);
+                var attribute = Assembly.GetExecutingAssembly().GetCustomAttribute<AssemblyTitleAttribute>();
```

Comment:
Warning: GetCustomAttribute<T> is a .NET 4.5 extension method.
This will break #124 again.

Reply:
> Could you elaborate on how we are breaking the plugin then?

Sorry not the plugin itself, but the users who are still using .NET 4.0 **AND** use the plgx file. 

> And if we cannot do anything about it, it would be nice that we know or are prevented from breaking stuff.

We could set the target version inside the csproj file to ``v4.0`` instead of ``v4.5``. 

> I'd love to have some things in place that make sure the plugin can be compiled.

Maybe we can fire up keepass instances on different .net frameworks and let them compile our plgx. But I don't know if github actions can work with GUI applications. 
Maybe we should add a feature-request to keepas to implement a ``--test-plgx`` command line parameter.

Reply:
> We could set the target version inside the csproj file to `v4.0` instead of `v4.5`.

I just tried to bump that, and that does not seem to work: the nuget packaged Keepass.exe is built against 4.5, although I guess Keepass itself does not really require 4.5. We would have to rework that part to make that work.

Given that 4.0 has been out since 2010, we could also bump the minimal version to something higher?

> Maybe we can fire up keepass instances on different .net frameworks and let them compile our plgx. But I don't know if github actions can work with GUI applications.
> Maybe we should add a feature-request to keepas to implement a `--test-plgx` command line parameter.

That is something I've been thinking about as well. Would indeed be really nice to have a command line option where Keepass does not fire up the GUI, but only compiles the plugin, checks compatibility and returns an exit code when things are wrong. For doing this in CI, I guess we could have some docker image.